### PR TITLE
NADUT

### PR DESCRIPTION
**Pull request by cptchris**
Added FIX ''NADUT''

### CODE REVIEW

File: Fixes.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 [
   {
     "Latitude": 0,
-    "Name": "0000N",
+    "Name": "000W",
```
Comment (Adrien-Z) [inline Fixes.json]: Why did you modified this name?

File: Fixes.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
     "Longitude": 0
+
```
Comment (Adrien-Z) [inline Fixes.json]: Remove this unnecessary line.

File: Fixes.json [hunk 3]
```diff
@@ -702870,9 +702871,14 @@
...
   {
+    "Latitude": 25.857500,
+    "Name": "NADUT",
+    "Longitude": 55.650000 
+  },
+  {
     "Latitude": 50.6196671,
     "Name": "NADUP",
-    "Longitude": -104.750336
+    "Longitude": -104.750336
```
Comment (Adrien-Z) [inline Fixes.json]: Remove that space after the longitude.

**Comment by cptchris** [comment]
Will do another pull request

**Pull request merged**